src/main/java/com/rackspace/ceres/app/web/MetadataController.java

```diff
@@ -95,6 +100,17 @@ public MetadataController(MetadataService metadataService, Environment environme
         .getTagValues(tenantHeader, metricName, tagKey);
   }
 
+  @GetMapping("/tags")
+  public Mono<TagKeyPairResponse> getTags(
+      @RequestParam(required = false) String metricName,
+      @RequestParam(required = false) String metricGroup,
+      @RequestHeader(value = "#{appProperties.tenantHeader}") String tenantHeader) {
```

Comment:
Add `@ApiIgnore` here so that this gets hidden from swagger. For reference you can take a look at other apis in this controller class.